www/content/build.md

```diff
@@ -149,3 +149,9 @@ GOVERSION=$(go version) goreleaser
  ```
 
  [hook]: /hooks
+
+## Define Build Tag
```

Comment:
I think we can describe both environments on the release page... or maybe on `environment.md` 🤔

Reply:
We can add it to `environment.md` but I would keep them separate because they impact different features. If there is a known issues / workarounds file, I would put it there also.

Reply:
hmm, yeah, makes sense... lets then add both to environment as well?

we can just link to the relevant topics 🤔

Reply:
Whichever you prefer :)

Reply:
lets do it then 🙏